src/test/java/de/tum/in/www1/artemis/exercise/textexercise/TextExerciseUtilService.java

```diff
@@ -356,8 +356,8 @@ public TextPlagiarismResult createTextPlagiarismResultForExercise(Exercise exerc
     }
 
     public TextAssessmentEvent createSingleTextAssessmentEvent(Long courseId, Long userId, Long exerciseId, Long participationId, Long submissionId) {
-        return TextExerciseFactory.generateTextAssessmentEvent(TextAssessmentEventType.VIEW_AUTOMATIC_SUGGESTION_ORIGIN, FeedbackType.AUTOMATIC, TextBlockType.AUTOMATIC, courseId,
-                userId, exerciseId, participationId, submissionId);
+        return TextExerciseFactory.generateTextAssessmentEvent(TextAssessmentEventType.EDIT_AUTOMATIC_FEEDBACK, FeedbackType.AUTOMATIC, TextBlockType.AUTOMATIC, courseId, userId,
+                exerciseId, participationId, submissionId);
```

Comment:
I was a bit confused at first, but this is just to generate any text assessment event for tests -> ok

Reply:
Yeah me too, but as far as I can see this is fine